tests/test.utils.jsx

```diff
@@ -71,3 +71,13 @@ describe('utils.isDisabled', function() {
   });
 
 });
+
+
+describe('utils.setTitle', function() {
+
+    it('should set the title', function() {
+        utils.setTitle('Pay', document);
```

Comment:
This is where you pass in your own object so that you don't manipulate the real title.